Rubberduck.Deployment/Rubberduck.Deployment.csproj

```diff
@@ -56,15 +55,7 @@
       <Output TaskParameter="Value" PropertyName="TargetAssembly" />
     </CreateProperty>
 
-    <RubberduckPostBuildTask 
-      Config="$(ConfigurationName)"
-      NetToolsDir="$(SdkPath)bin\NETFX 4.6.1 Tools\"
-      WixToolsDir="$(ProjectDir)WixToolset\"
-      SourceDir="$(TargetDir)"
-      TargetDir="$(TargetDir)"
-      ProjectDir="$(ProjectDir)"
-      IncludeDir="$(ProjectDir)InnoSetup\Includes\"
-      FilesToExtract="Rubberduck.dll|Rubberduck.API.dll" />
+    <RubberduckPostBuildTask Config="$(ConfigurationName)" NetToolsDir="$(SdkPath)bin\NETFX 4.6.1 Tools\" WixToolsDir="$(ProjectDir)WixToolset\" SourceDir="$(TargetDir)" TargetDir="$(TargetDir)" ProjectDir="$(ProjectDir)" IncludeDir="$(ProjectDir)InnoSetup\Includes\" FilesToExtract="Rubberduck.dll|Rubberduck.API.dll" />
```

Comment:
Why the formatting change?

Anyway, this still states that the API should be extracted.

Reply:
Huh, that wasn't me... gosh and there I was looking for where the `RubberduckPostBuildTask` came from!
Thanks!